caravel/assets/javascripts/explorev2/stores/store.js

```diff
@@ -1630,6 +1630,7 @@ const defaultFormData = {};
 defaultFormData.slice_name = null;
 defaultFormData.slice_id = null;
 Object.keys(fields).forEach((k) => { defaultFormData[k] = fields[k].default; });
+exports.defaultFormData = defaultFormData;
```

Comment:
it would be nice to use the 
`export function thing() {}` or `export const thing = {}` syntax here, rather than `exports.thing = thing;`

maybe `defaultFormData` should be a function like this:

```js
export function defaultFormData() {
  const data = {
    slice_name: null,
    slice_id: null,
  };
  Object.keys(fields).forEach((k) => { data[k] = fields[k].default; });
  return data;
}
```